core-java-modules/core-java-concurrency-advanced-4/src/main/java/com/baeldung/threadparking/Application.java

```diff
@@ -0,0 +1,25 @@
+package com.baeldung.threadparking;
+
+import java.util.concurrent.locks.LockSupport;
+
+public class Application {
+
+    public static void main(String[] args) throws InterruptedException {
```

Comment:
I think the same way Object.wait() can throw the InterruptedException, the park method cannot. Could you review it?

Reply:
The reason the exception is there is that it is mandatory for the `Thread.sleep()` method which is however, commented out. In the tutorial, where the reader will go through step by step, at some point the `main()` method will have declare the thrown exception.